### ISSUE TITLE
'Codelabs' page issue

### ISSUE THREAD

**Issue by mu-us61** [flutter/website#5811]
Page URL: https://codelabs.developers.google.com/codelabs/flutter#0

Page source: https://github.com/flutter/website/tree/master/src/docs/codelabs/index.md

Found a typo? You can fix it yourself by going to the page source and clicking the pencil icon. Or finish creating this issue.

Description of issue:

No " next " button for mobile web view, so it stuck

This is a normal working Desktop version ;
![image](https://user-images.githubusercontent.com/73366454/118592173-35383280-b7ae-11eb-8c7a-e93e89c31370.png)

This is same page on mobile and cant go to next page , the button is actually there but too down, only little blue is there.. and completely cant see in some mobile browsers like opera mobile ;

![image](https://user-images.githubusercontent.com/73366454/118592258-5e58c300-b7ae-11eb-9d4b-8d0ca6d49c0a.png)

![image](https://user-images.githubusercontent.com/73366454/118592683-07072280-b7af-11eb-9e66-623a677eddfd.png)

**Comment by micpap25** [comment]
It looks like you're on Android. I can confirm that this happens on my iPhone as well. 
This seems like an external issue for Codelabs so this issue might be on hold for a little bit.

**Comment by domesticmouse** [comment]
Blocked on https://github.com/googlecodelabs/tools/issues/575

**Comment by parlough** [comment]
The next button seems to be floating now when I test this and other codelabs on a mobile device.

I'll close this, but if anyone experiences a similar issue again, let us know. Thanks!
